css/Styles.css

```diff
@@ -1,28 +1,485 @@
+* {
+    padding: 0px;
+    margin: 0px;
+}
+
+body {
+    background-color: rgb(247, 247, 247);
+    font-family:"Work Sans", sans-serif;
+}
+
+nav {
+    max-width: 110%;
```

Comment:
Should never be 100%, what are we trying to achieve? Can this be done any other way?